src/dynarec/rv64/dynarec_rv64_helper.c

```diff
@@ -994,18 +994,19 @@ int x87_setround(dynarec_rv64_t* dyn, int ninst, int s1, int s2, int s3)
     // MMX/x87 Round mode: 0..3: Nearest, Down, Up, Chop
     // RV64: 0..7: Nearest, Toward Zero (Chop), Down, Up, Nearest tie to Max, invalid, invalid, dynamic (invalid here)
     // 0->0, 1->2, 2->3, 3->1
-    SLLI(s1, s1, 1);
```

Comment:
I can't figure out what the original version is trying to do, and it's problematic, so I replaced it with a more trivial version (which has three direct jumps).

Reply:
Well, it's trying to change from 0, 1, 2, 3 to 0, 2, 3, 1 as the last comment says